bin/lux

```diff
@@ -39,6 +39,14 @@ function commandNotFound(cmd) {
   console.log(`  Use ${green('lux --help')} for a full list of commands${EOL}`);
 }
 
+function setEnvVar(key, val, def) {
+  if (val) {
+    process.env[key] = val;
+  } else if (!process.env[key]) {
+    process.env[key] = def;
+  }
+}
+
```

Comment:
This really doesn't make a difference since we know where the `key` and `val` arguments are coming from but for consistency can we modify this to use the [`Reflect`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Reflect) API here?

```javascript
function setEnvVar(key, val, def) {
  if (val) {
    Reflect.set(process.env, key, val);
  } else if (!Reflect.has(process.env, key)) {
    Reflect.set(process.env, key, def);
  }
}
```